extensions/amp-autocomplete/0.1/amp-autocomplete.css

```diff
@@ -43,6 +43,12 @@ amp-autocomplete > input {
   z-index: 10;
 }
 
+.i-amphtml-autocomplete-results-up {
+  top: auto;
+  bottom: 100%;
+  margin-bottom: .5rem; /* default-ui-space-small */
```

Comment:
Can you help me understand the use of root-ems here? As I understand it, as written this margin will vary with the font size. /cc @andrewwatterson

Reply:
I think `rem` is the appropriate unit here, but I could be convinced. The question I usually ask is "if the font size were huge/tiny, would we want this value to scale?" Sometimes it's really clear: `line-height` should always scale with font size; the size of an icon next to some text probably should as well. Margins are a little fuzzier, and especially since this isn't even a margin directly around text, keeping it as a fixed size seems reez.